arudnicenco/README.md

```diff
@@ -0,0 +1,107 @@
+# Creating tables 
+
+``` sql
+CREATE TABLE teachers (
+teacher_id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+teacher_fname VARCHAR(25) NOT NULL,
+teacher_lname VARCHAR(25) NOT NULL
+);
+```
+
+``` sql
+CREATE TABLE students (
+student_id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+student_fname VARCHAR(25) NOT NULL,
+student_lname VARCHAR(25) NOT NULL
+);
+ ```
+
+``` sql
+CREATE TABLE subjects (
+subject_id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+subject_name VARCHAR(25) NOT NULL,
+teacher_id INT NOT NULL,
+CONSTRAINT FOREIGN KEY (teacher_id) REFERENCES teachers(teacher_id)
+);
+```
+
+``` sql
+CREATE TABLE marks (
+subject_id INT NOT NULL,
+student_id INT NOT NULL,
+mark INT NOT NULL,
+CONSTRAINT FOREIGN KEY (subject_id) REFERENCES subjects(subject_id),
+CONSTRAINT FOREIGN KEY (student_id) REFERENCES students(student_id)
+);
+```
+
+``` sql
+CREATE TABLE faculties ( 
+faculty_id INT NOT NULL PRIMARY KEY AUTO_INCREMENT, 
+faculty_name VARCHAR(25) 
+);
+```
+``` sql
+CREATE TABLE groups (
+group_id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+group_name VARCHAR(25) NOT NULL,
+student_id INT NOT NULL,

```

Comment:
Relatia Student - Grupa trebuie detinuta de catre student deoarece este multi la unul (multi studenti - o grupa). In cazul de fata reiese ca un student are mai multe grupe, iar o grupa - doar un student.